test/e2e/e2eslow/backup_restore_test.go

```diff
@@ -78,6 +83,7 @@ func TestBackupAndRestore(t *testing.T) {
 	s3Path := path.Join(os.Getenv("TEST_S3_BUCKET"), "jenkins", suffix, time.Now().Format(time.RFC3339), "etcd.backup")
 
 	testEtcdBackupOperatorForS3Backup(t, clusterName, operatorClientTLSSecret, s3Path)
+	testEtcdBackupOperatorForPeriodicS3Backup(t, clusterName, operatorClientTLSSecret, s3Path)
```

Comment:
@ukinau I think I know why the test is failing. Previously we had the two subtests:
```Go
testEtcdBackupOperatorForS3Backup(t, clusterName, operatorClientTLSSecret, s3Path)
testEtcdRestoreOperatorForS3Source(t, clusterName, s3Path)
```
The first test creates the backup file at `s3Path` and the second expects the backup file to be present in order to restore a cluster.

With the periodic backup sub test now in between the two we are now deleting all the backup files in a defer statement. So `testEtcdRestoreOperatorForS3Source` will fail because the previous test deleted the backup file.
```Go
func testEtcdBackupOperatorForPeriodicS3Backup(t, clusterName, operatorClientTLSSecret, s3Path) {
        ...
        // create etcdbackup resource
	eb, err := f.CRClient.EtcdV1beta2().EtcdBackups(f.Namespace).Create(backupCR)
	if err != nil {
		t.Fatalf("failed to create etcd back cr: %v", err)
	}
	defer func() {
		if err := f.CRClient.EtcdV1beta2().EtcdBackups(f.Namespace).Delete(eb.Name, nil); err != nil {
			t.Fatalf("failed to delete etcd backup cr: %v", err)
		}
		// cleanup backup files
		allBackups, err = wr.List(context.Background(), backupS3Source.Path)
		if err != nil {
			t.Fatalf("failed to list backup files: %v", err)
		}
		if err := e2eutil.DeleteBackupFiles(wr, allBackups); err != nil {
			t.Fatalf("failed to cleanup backup files: %v", err)
		}
	}()
        ...
}
```

Can you please move `testEtcdBackupOperatorForPeriodicS3Backup` at the end so it doesn't interfere with the restore test.

```Go
// Backup then restore tests
testEtcdBackupOperatorForS3Backup(t, clusterName, operatorClientTLSSecret, s3Path)
testEtcdRestoreOperatorForS3Source(t, clusterName, s3Path)
// Periodic backup test
testEtcdBackupOperatorForPeriodicS3Backup(t, clusterName, operatorClientTLSSecret, s3Path)
```